### ISSUE TITLE
Regular Expression Denial of Service

### ISSUE THREAD

**Issue by jamesarosen** [isaacs/minimatch#118]
npm now includes some security warnings that print on each install. For my application,
```
$ npm i
updated 1 package and moved 1 package in 24.608s
[!] 192 vulnerabilities found [270889 packages audited]
    Severity: 20 Low | 16 Moderate | 156 High | 0 Critical
    Run `npm audit` for more detail
```
Running `npm audit | grep High -a 5` shows all High vulnerabilities in my application are "Regular Expression Denial of Service" from minimatch. I'd love to get these shored up.

**Comment by jamesarosen** [comment]
It looks like this was patched in 3.0.2.

**Comment by camdagr8** [comment]
Was this patched in 3.0.2? I have 3.0.2 as a dep. and still get the audit vulnerabilities.

**Comment by isaacs** [comment]
@camdagr8 Could it be that you are using a dep that depends on an earlier version?  The `npm audit` output should tell you where in the tree it lives.  You can also run `npm ls minimatch` to see all versions in your tree.
